tools/ete/ete_lineage_generator.xml

```diff
@@ -0,0 +1,143 @@
+<tool id="ete_lineage_generator" name="ETE lineage generator" version="3.0.0b35">
+		<description>from a list of species/taxids using the ETE Toolkit</description>
+     <requirements>
+        <requirement type="package" version="3.0.0b35">ete3</requirement>
+    </requirements>
+    <stdio>
+        <!-- Anything other than zero is an error -->
+        <exit_code range="1:" />
+    </stdio>
+    <command>
```

Comment:
You can add `detect_errors="exit_code"` attribute to `<command>` and remove the whole `<stdio>` block.